tests/platform/windows/windows-jaws-testSpec.txt

```diff
@@ -5,6 +5,6 @@ This will run the acceptance tests for the JAWS screenreader (Freedom Scientific
 
 Prerequisites:
 * JAWS needs to be installed
-* In JAWS, a profile should be created by going to "options"->voices->Voice Adjustment. Make sure that "Eloquence - (factory)" is selected in the profile name. Click "Save As.." button and write "cloud4all" as the name (no quotes). Once this is done, Jaws should be working properly
+* In JAWS, a profile should be created by going to "options"->voices->Voice Adjustment. Make sure that "Eloquence - (factory)" is selected in the profile name. Click "Save As.." button and write "GPII" as the name (no quotes). Once this is done, Jaws should be working properly
 
```

Comment:
Would you happen to know of any way to do this automatically. Is there some sort of wizard or the like, that would allow us to create a new voice profile automatically via the GPII? Most important is that we would really want to be able to create this profile without requiring any user interaction. So the when the GPII wanted to run JAWS, it would check if there is a GPII voice profile, and if not, it would automaticlly create it.

There might also be alternative ways of setting the voice settings, without requiring to create a new voice profile. Basically, anything that would allow us to edit the Jaws settings without first have to manually do some steps would be extremely useful.